### ISSUE TITLE
ag-grid ADA support

### ISSUE THREAD

**Issue by MorMeshulam** [ag-grid/ag-grid#3894]
After my site have been checked by accessibility advisor company,
we having some issues.

**Current behaviour**

**1. The reading order (as determined by code order) is not logical or intuitive.**

**2. There is a keyboard trap from the second row  column one of the tables in shift tab navigation**

**3. The data table has missing header cell markup.
i.e, Column header of the Table are not marked programmatically  as a result the association is not exposed to the screen reader users**

**4. Table is incorrectly exposed to the screen reader users
Currently, Screen reader reads as 
 table  with 1 rows and 4 columns** 

**Expected behaviour**

I expect the grid columns header to have the right aria attributes and roles
I expect the grid to render correct with the right order 
i expect  the tab order to be correct and consistent.

**Please tell us about your environment:**

1. linux env
2. reactjs with webpack
3. python

* **ag-Grid version:** X.X.X
   
"ag-grid-'community": "^23.1.1",
"ag-grid-react": "^23.1.1",

* **Browser:**

4. chrome

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
english

**Comment by makinggoodsoftware** [comment]
Hi,

ag-grid staff does not provide support through github. If you are a customer please contact us through our official support channel. Note that this issue has been flagged as managed-by-the-community, we also recommend you to check stackoverflow, note that if there is no activity on this issue after a while we will close it.

Thanks
